cmd/check_container.go

```diff
@@ -161,7 +161,11 @@ func getContainerPolicyExceptions(ctx context.Context, pc pyxisClient) (policy.P
 	}
 	log.Debugf("Certification project name is: %s", certProject.Name)
 	if certProject.Container.Type == "scratch" {
-		return policy.PolicyScratch, nil
+		if certProject.Container.Privileged {
+			return policy.PolicyScratchRoot, nil
+		} else {
+			return policy.PolicyScratchNonRoot, nil
+		}
```

Comment:
```suggestion
		if certProject.Container.Privileged {
			return policy.PolicyScratchRoot, nil
		}
		
		return policy.PolicyScratchNonRoot, nil
		
```
We can remove the else and just return `nonroot` as the default

Reply:
@acornett21 Thanks a lot. Done.